src/transformers/pipelines.py

```diff
@@ -614,7 +636,7 @@ def __call__(
 
             result = []
             for generated_sequence in output_sequences:
-                generated_sequence = generated_sequence.tolist()
+                generated_sequence = generated_sequence.numpy().tolist()
```

Comment:
TF does not have a `.tolist()` function on tensors...

Reply:
This might have broken generation for PyTorch? My code that runs on 2.9.0 now says "can't convert CUDA tensor to numpy. Use Tensor.cpu() to copy the tensor to host memory first."

Reply:
Yes `.tolist()` in PyTorch also moves to cpu under the hood. Unless TF has a .cpu() method on tensors, we'll have to add an `if` clause on the framework @patrickvonplaten